### PR TITLE
example function added for call by reference

### PR DESCRIPTION
**Pull request by himanshu81494**
added swap function as an example for call by reference.
also added usage of "goto" in C.

### CODE REVIEW

File: c.html.markdown [hunk 1]
```diff
@@ -472,7 +491,22 @@ char c[] = "This is a test.";
 str_reverse(c);
 printf("%s\n", c); // => ".tset a si sihT"
 */
-
+//as we can return return only one variable
```
Comment (levibostian) [inline c.html.markdown]: `return return` => `return`

File: c.html.markdown [hunk 2]
```diff
@@ -472,7 +491,22 @@ char c[] = "This is a test.";
...
+//to change values of more than one variables we use call by reference
+void swapTwoNumbers(int *a, int *b)
+{
+int temp = *a;
```
Comment (levibostian) [inline c.html.markdown]: indent these, please to keep with style of rest of doc.

File: c.html.markdown [hunk 3]
```diff
@@ -306,7 +307,25 @@ int main(void) {
...
     break;
   }
-
+  
+  // using "goto"
+  typedef enum { false, true } bool;
+  // for C don't have bool as data type :(
+  bool disaster = false;
+  int i, j;
+  for(i=0;i<100;++i)
```
Comment (levibostian) [inline c.html.markdown]: Two for loops stacked on top of each other, this is possible?

File: c.html.markdown (hunk 1 above)
Comment (himanshu81494) [inline c.html.markdown]: sorry i did not  understand what "return return => return " means.

File: c.html.markdown (hunk 3 above)
Comment (himanshu81494) [inline c.html.markdown]: https://ideone.com/GuPhd6

please correct me if I am somewhere wrong. This is my first ever pull request on git !

File: c.html.markdown (hunk 1 above)
Comment (vendethiel) [inline c.html.markdown]: you have two times the "return" word

File: c.html.markdown (hunk 2 above)
Comment (himanshu81494) [inline c.html.markdown]: OK, I did it.

File: c.html.markdown (hunk 1 above)
Comment (himanshu81494) [inline c.html.markdown]: got it 
I have corrected it.

File: c.html.markdown (hunk 3 above)
Comment (levibostian) [inline c.html.markdown]: No, you are not mistaken. I don't know why I thought it was not possible, it is just nested loops not that weird. I had just never seen it done like this, looked weird or something.

**Comment by levibostian** [comment]
Thanks!

**Pull request merged**